src/dos/cdrom_image.cpp

```diff
@@ -835,8 +835,8 @@ void CDROM_Interface_Image::ChannelControl(TCtrl ctrl)
 	}
 
 	// Adjust the volume of our mixer channel as defined by the application
-	player.channel->SetVolumeScale(static_cast<float>(ctrl.vol[0] / 255.0), // left vol
-	                               static_cast<float>(ctrl.vol[1] / 255.0)); // right vol
+	player.channel->SetAppVolume(static_cast<float>(ctrl.vol[0] / 255.0), // left vol
+	                             static_cast<float>(ctrl.vol[1] / 255.0)); // right vol
 
```

Comment:
`255.0f` to avoid double-to-float conversion

Reply:
Fixed.